packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -144,6 +144,14 @@ export const GridMixin = (superClass) =>
           reflectToAttribute: true,
         },
 
+        /**
+         * String used to label the grid to screen reader users.
+         * @attr {string} accessible-name
+         */
+        accessibleName: {
```

Comment:
Maybe this should be in `vaadin-grid-a11y-mixin`?

Reply:
Right. I missed that mixin. Done.